services/distribution/src/main/java/app/coronawarn/server/services/distribution/dgc/DigitalGreenCertificateToProtobufMapping.java

```diff
@@ -45,7 +45,7 @@ VaccineJsonStringObject readMahJson() {
    *
    * @return The corresponding JSON object.
    */
-  VaccineJsonStringObject readMedicinalProductJson() {
+  VaccineJsonStringObject readMedicinalProductJson() throws DefaultValuesetsMissingException {
```

Comment:
Is it DefaultValuesetsMissingException or DefaultValueSetsMissingException?

Reply:
Wrong camel casing...